src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -37,6 +46,33 @@
 from system import new_process
 from system import shell
 
+# Maximum length of a random chosen length for `-max_len`.
+MAX_VALUE_FOR_MAX_LENGTH = 10000
+
+# Probability of doing DFT-based fuzzing (depends on DFSan build presence).
+DATAFLOW_TRACING_PROBABILITY = 0.25
+
+# Prefix for the fuzzer's full name.
+LIBFUZZER_PREFIX = 'libfuzzer_'
+
+# Allow 30 minutes to merge the testcases back into the corpus.
+DEFAULT_MERGE_TIMEOUT = 30 * 60
+
+MERGED_DICT_SUFFIX = '.merged'
+
+MERGE_DIRECTORY_NAME = 'merge-corpus'
+
+HEXDIGITS_SET = set(string.hexdigits)
```

Comment:
just use in-place, rather than here. (only one occurence).

Reply:
Done!